src/examples/accordions/stepper/StepperLayout.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Stepper } from '@zendeskgarden/react-accordions';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledButtons = styled.div`
+  margin-top: ${p => p.theme.space.sm};
+  padding: 3px;
+
+  & > button {
```

Comment:
Can this be `& > ${Button}`?

Reply:
When I try to target it that way, my build doesn't succeed. It doesn't outright fail in my terminal, but in the browser it doesn't complete. This is the error message:

```
Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component.
```

Reply:
wow. Ok, good for now. Thx.